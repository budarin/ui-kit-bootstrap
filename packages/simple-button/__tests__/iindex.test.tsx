// import * as React from 'react';
import { shallow } from 'enzyme';
import Button from '../src/index';

test('Test Button component', () => {
    // const btn = shallow(<Button text="Off" />);

    // Interaction demo
    // expect(btn.text()).toEqual('Off');
    // btn.find('input').simulate('change');
    // expect(btn.text()).toEqual('On');

    // Snapshot demo
    // expect(btn).toMatchSnapshot();

    expect(0).toBe(0);
});
