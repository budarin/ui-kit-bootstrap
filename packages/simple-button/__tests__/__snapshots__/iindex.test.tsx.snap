// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Кнопка должна содержать текст "Button"" 1`] = `
<button
  className="btn"
>
  <img
    alt=""
    src="test-file-stub"
  />
  <span
    style={
      Object {
        "fontStyle": "italic",
      }
    }
  >
    Button
  </span>
</button>
`;

exports[`Кнопка должна содержать текст по-умолчанию 1`] = `
<button
  className="btn"
>
  <img
    alt=""
    src="test-file-stub"
  />
  <span
    style={
      Object {
        "fontStyle": "italic",
      }
    }
  >
    Simple button
  </span>
</button>
`;
